src/logion/services/locauthorization.service.ts

```diff
@@ -25,14 +25,22 @@ export class LocAuthorizationService {
         }
     }
 
-    async isContributor(request: LocRequestAggregateRoot, contributor: AuthenticatedUser): Promise<boolean> {
+    async isContributor(request: LocRequestAggregateRoot, contributor: AuthenticatedUser, allowOwner = true): Promise<boolean> {
         return (
-            accountEquals(request.getOwner(), contributor) ||
+            (accountEquals(request.getOwner(), contributor) && this.ownerCanContributeItems(request, allowOwner)) ||
             accountEquals(request.getRequester(), contributor) ||
             await this.isSelectedIssuer(request, contributor)
         );
     }
 
+    private ownerCanContributeItems(request: LocRequestAggregateRoot, allowOwner: boolean): boolean {
```

Comment:
This logic could be moved to the VO I mention above.

Reply:
Done in [#749db6f](https://github.com/logion-network/logion-backend-ts/pull/295/commits/749db6f421d6f8da03b6a53513fc6146ec43ef59)